### DISCUSSION TITLE
Prompt dialog box transparent because of my small, undetected bug in qutebrowser config

### DISCUSSION THREAD

**Discussion by ashwinvis** [qutebrowser/qutebrowser#6709] [Show and tell] [answered]
# Context

In config.py I have a `_font_size` variable to scale up the UI. 

```py
...

_font_size = '16px'  # default 10pt

# Font used in the completion widget.
# Type: Font
c.fonts.completion.entry = '%s monospace' % _font_size

# Font used in the completion categories.
# Type: Font
c.fonts.completion.category = 'bold %s monospace' % _font_size

...

# Font used for prompts.
# Type: Font
c.fonts.prompts = '%s sans-serif'  # NOTE: I missed to add the `% _font_size` part

...

```

This caused the prompt to be invisible. For example, if I open reddit.com, I get this:

![Screenshot_20210927_160654](https://user-images.githubusercontent.com/9155111/134928257-610e8978-1331-405c-8be1-a6a485a3355b.png)

I don't think this counts as a qutebrowser bug since I had a bug in my config, but it was fun tracking it down.

**Comment by The-Compiler** [comment, marked as answer]
Yeah, unfortunately Qt doesn't have a straightforward way for us to validate whether those settings are valid. We're trying to validate them to catch the most common issues, but the validation for fonts is rather lax (because a lot of things can be part of a font name...).

I added it to #115, also see #2571.
